controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/PathGroup.java

```diff
@@ -153,7 +156,8 @@
                          "/application/v4/tenant/{tenant}/application/{application}/environment/test/region/{region}/instance/{ignored}",
                          "/application/v4/tenant/{tenant}/application/{application}/environment/test/region/{region}/instance/{ignored}/deploy",
                          "/application/v4/tenant/{tenant}/application/{application}/environment/staging/region/{region}/instance/{ignored}",
-                         "/application/v4/tenant/{tenant}/application/{application}/environment/staging/region/{region}/instance/{ignored}/deploy"),
+                         "/application/v4/tenant/{tenant}/application/{application}/environment/staging/region/{region}/instance/{ignored}/deploy",
+                         "/routing/v1/inactive/tenant/{tenant}/application/{application}/instance/{ignored}/environment/prod/region/{region}"),
```

Comment:
Ah, there was one `/inactive/` as well? So I guess this is to allow application owners to manipulate endpoint status? The `athenzTenantAdmin` does not cover this group — it was split out specifically for the legacy pipeline. Putting this under the `applicationInfo` should work for both sets of roles (Athenz and otherwise).

Reply:
Done. There's too many groups!

Reply:
Yes, should be able to remove some already now.